source/blog/2015-06-18-ember-data-1-13-released.md

```diff
@@ -0,0 +1,578 @@
+JSON-API adapter and serializer will be the default adapter loaded by
+Ember Data.
+
+### New Serializer API
+
+The internal data format that Ember Data's expects its serializers to
+return has evolved over the 3.5 years of the project. Unfortunately,
+this has resulted in some strange gotcha's around use cases like meta
+data, polymorphic relationships and relationship links, especially
+when these features are combined. Another problem with Ember Data's
+internal format has been its lack of documentation which has made it
+has made it hard to create custom serializers for Ember Data without
+already having a strong understanding of the Ember Data codebase.
+
+In order to solve these problems Ember Data is fully embracing
+JSON-API. `store.push` now takes a JSON-API document and serializers
+are expected to return a JSON-API document when normalizing the
+adapter response. JSON-API solves many of the gotchas Ember Data's
+internal format has struggles with over the years and it is a
+[well documented format](http://jsonapi.org/format/) that serializer
+authors can target without having to know about Ember Data's
+internals.
+
+Ember Data 1.13 supports the old format (with deprecations) and the
+new JSON-API format to make it easy to transition.
+
+#### Transition to the new Serializer API
+
+If you have a custom serializer you will need to make some new changes
+to your serializer to get it ready for Ember Data 2.0.
+
+#### Custom extract methods
+
+If you have custom `extract` hooks you need to do two things:
+
+Rename:
+
+- `extract()              => normalizeResponse()`
+- `extractFindAll()       => normalizeFindAllResponse()`
+- `extractFind()          => normalizeFindRecordResponse()`
+- `extractFindMany()      => normalizeFindManyResponse()`
+- `extractFindBelongsTo() => normalizeFindBelongsToResponse()`
+- `extractFindHasMany()   => normalizeFindHasManyResponse()`
+- `extractFindQuery()     => normalizeQueryResponse()`
+- `extractQueryRecord()   => normalizeQueryRecordResponse()`
+- `extractCreateRecord()  => normalizeCreateRecordResponse()`
+- `extractDeleteRecord()  => normalizeDeleteRecordResponse()`
+- `extractUpdateRecord()  => normalizeUpdateRecordResponse()`
+- `extractSave()          => normalizeSaveResponse()`
+- `extractSingle()        => normalizeSingleResponse()`
+- `extractArray()         => normalizeArrayResponse()`
+
+Make sure that your `extract` hooks return a full JSON-API Document
+([http://jsonapi.org/format/#document-top-level]()) with the primary
+resource(s) in a `data` array or object and eventual sideloaded
+resources in a `included` array. Every resource object
+([http://jsonapi.org/format/#document-resource-objects]()) has to
+follow the JSON-API format with `id`, `type`, attributes in
+`attributes` and relationships in `relationships`. For the full
+specification of the JSON-API format see
+[http://jsonapi.org/format/]().
+
+##### Custom extractMeta method If you have a custom `extractMeta`
```

Comment:
Why headlines for these three items? The capitalization and sentence structures seems really whack. `Custom extractMeta method If you have` for example. Seems half-done